cms/djangoapps/contentstore/views/item.py

```diff
@@ -846,7 +849,14 @@ def create_xblock_info(xblock, data=None, metadata=None, include_ancestor_info=F
         "course_graders": json.dumps([grader.get('type') for grader in graders]),
         "has_changes": has_changes,
         "actions": xblock_actions,
-        "explanatory_message": explanatory_message
+        "explanatory_message": explanatory_message,
+        "enable_proctored_exams": enable_proctored_exams,
```

Comment:
I'm not 100% clear on the recommendation. 

Are you saying follow this patter:

```
    if include_ancestor_info:
        xblock_info['ancestor_info'] = _create_xblock_ancestor_info(xblock, course_outline)
```
?